build.sbt

```diff
@@ -70,7 +73,7 @@ lazy val app =
     .settings(vcpkgNativeConfig())
     .settings(
       scalaVersion := Versions.Scala,
-      vcpkgDependencies := Set("libpq", "openssl", "libidn2"),
```

Comment:
Did you mean to remove libidn2?

Reply:
No, was to make my local setup easier but I pushed it by mistake. Going to revert it.